cellprofiler/utilities/sharedmethods.py

```diff
@@ -0,0 +1,33 @@
+"""
+Shared methods used by multiple modules
```

Comment:
Let's create functionality-specific modules. A `cellprofiler.utilities.morphology` module would be specific.